generators/app/index.js

```diff
@@ -95,6 +97,31 @@ module.exports = generators.Base.extend({
       }.bind(this));
     },
 
+    askForGithubAccount: function () {
+      if (this.pkg.repository) {
+        this.props.githubAccount = this.pkg.repository.split('/')[0];
```

Comment:
Repository can be full urls and they can point elsewhere than github.

This line needs to be smarter.